hashicorp/dev-portal
# refactor: use MdxBlockquote

## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Moves `blockquote` styles from `dev-dot-content` CSS into an MDX custom component.

## 🤷 Why

- To avoid the change of nested global selectors leaking into unexpected areas of MDX custom components
- To consolidate MDX styling into MDX custom components

## 🧪 Testing

- Visit a docs MDX page, such as [/vault/docs/commands/operator/usage][/vault/docs/commands/operator/usage], and confirm the blockquote on the page looks as expected (vs [upstream][UPSTREAM/vault/docs/commands/operator/usage])
- Visit a tutorial MDX page, such as [/vault/tutorials/getting-started/getting-started-dynamic-secrets][/vault/tutorials/getting-started/getting-started-dynamic-secrets], and confirm the blockquote on the page looks as expected (vs [upstream][UPSTREAM/vault/tutorials/getting-started/getting-started-dynamic-secrets])

## 💭 Anything else?

Not at the moment!

[task]: https://app.asana.com/0/1202097197789424/1202584446902873/f
[preview]: https://dev-portal-git-zsmdx-blockquote-hashicorp.vercel.app
[/vault/docs/commands/operator/usage]: https://dev-portal-git-zsmdx-headings-hashicorp.vercel.app/vault/docs/commands/operator/usage
[UPSTREAM/vault/docs/commands/operator/usage]: https://developer.hashicorp.com/vault/docs/commands/operator/usage
[/vault/tutorials/getting-started/getting-started-dynamic-secrets]: https://dev-portal-git-zsmdx-headings-hashicorp.vercel.app/vault/tutorials/getting-started/getting-started-dynamic-secrets
[UPSTREAM/vault/tutorials/getting-started/getting-started-dynamic-secrets]: https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-dynamic-secrets
